### ISSUE TITLE
The closing indent of closure has changed from 0.50.7.

### ISSUE THREAD

**Issue by bookii** [nicklockwood/SwiftFormat#1342]
I have the following code:

```swift
import UIKit

UIView.animateKeyframes(withDuration: 0.2, delay: 0, options: [], animations: {
    UIView.addKeyframe(withRelativeStartTime: 0,
                       relativeDuration: 0.1, animations: {
                           // ...
                       })
}) // <--- here
```

When I run swiftformat 0.50.7:

```swift
import UIKit

UIView.animateKeyframes(withDuration: 0.2, delay: 0, options: [], animations: {
    UIView.addKeyframe(withRelativeStartTime: 0,
                       relativeDuration: 0.1, animations: {
                           // ...
                       })
    }) // <--- here
```

This is undesirable.
If this is a bug, I would like to see it reverted.

**Comment by jshier** [comment]
I'm also seeing this.

```swift
return IdentifiedArrayOf(uniqueElements: values.map { id, amount, date in
    Expense(id: String(id),
            approvedBy: .lauren,
            amount: Decimal(amount),
            client: "Test Client",
            reimbursement: benefit,
            submittedDate: date,
            expenseDescription: "A description.")
    })
```

**Comment by ju135** [comment]
I'm also having this issue

**Comment by nicklockwood** [comment]
@bookii @jshier @ju135 fixed in 0.50.8
